### ISSUE TITLE
Thank you

### ISSUE THREAD

**Issue by linuxgurugamer** [RayKV423/Star-Trek-for-HP-2000#1]
This brought back memories, my first exposure to a computer was the HP 2000 in 1972, on a Teletype 33 with attached paper tape.

Thank you for preserving this

**Comment by RayKV423** [comment]
> This brought back memories, my first exposure to a computer was the HP 2000 in 1972, on a Teletype 33 with attached paper tape.
> 
> Thank you for preserving this

No problem! 

Whoa, you saw the actual paper tape of the Star Trek itself? Back in 1972? Wow, that’s pretty cool.

**Comment by linuxgurugamer** [comment]
Yes.  I don't remember the details.  I have an old Teletype sitting here in my office as a sort of memior.

**Comment by RayKV423** [comment]
> Yes. I don't remember the details. I have an old Teletype sitting here in my office as a sort of memior.

Very cool. My old library teacher have an electric typewriter in her office, looks pretty old, maybe 70s-80s? Pretty sure it still works. 

I hear the teletype is how you made tapes or something?

**Comment by linuxgurugamer** [comment]
Yes, the paper punch is on the left side.  Here is a video showing a paper tape being read:
https://youtu.be/CiUXDHtPnWY

**Comment by RayKV423** [comment]
> This brought back memories, my first exposure to a computer was the HP 2000 in 1972, on a Teletype 33 with attached paper tape.
> 
> Thank you for preserving this

Oh I should’ve asked this before but was the there any difference between the original version and the earliest surviving code I posted here? Or were they entirely the same?
